ed25519/Cargo.toml

```diff
@@ -13,9 +13,7 @@ parity-bytes = { git = "https://github.com/paritytech/parity-common" }
 ed25519_verify = { path = "ed25519_verify" }
 sha2 = "0.7.1"
 hex = "0.3.1"
-
-[dependencies.ed25519-dalek]
-version = "^0.8"
+ed25519-dalek = { git = "https://github.com/jwasinger/ed25519-dalek", branch="bad-line" }
```

Comment:
Instead of forking to disable a feature, consider setting the feature list here: https://github.com/jwasinger/ed25519-dalek#features

Reply:
Doesn't build:
```
error[E0635]: unknown feature `rand`
   --> /home/ewasm/.cargo/registry/src/github.com-1ecc6299db9ec823/ed25519-dalek-0.8.0/src/lib.rs:257:42
    |
257 | #![cfg_attr(feature = "nightly", feature(rand))]
    |                                          ^^^^
```

@axic do you know what is causing this?  I have no clue and googling for rust+wasm32 problems is a crapshoot.

Reply:
huh.. is this because `rand` doesn't really make sense in a pure wasm environment.